CONTRIBUTING.md

```diff
@@ -420,6 +427,30 @@ $ ./node ./test/parallel/test-stream2-transform.js
 Remember to recompile with `make -j4` in between test runs if you change code in
 the `lib` or `src` directories.
 
+##### Test Coverage
+
+It's good practice to try to improve test coverage, as you add new features
+or work on an existing subsystems.
+
+To run the test suite with test coverage:
+
+```text
+$ ./configure --coverage && make coverage
+```
+
+A detailed coverage report will be output to `coverage/index.html` for
+JavaScript coverage and to `coverage/cxxcoverage.html` for C++ coverage.
+
+To collect coverage for a subset of tests you can set the `CI_JS_SUITES` and
+`CI_NATIVE_SUITES` variables:
+
+```text
+$ CI_JS_SUITES=suites_child-process CI_NATIVE_SUITES= make coverage
```

Comment:
`$ make coverage CI_JS_SUITES=suites_child-process CI_NATIVE_SUITES=` - also works with shells that don't support `VAR=value cmd` syntax.

Reply:
I didn't know they existed, which shells are those?

Reply:
At least `csh` and `tcsh` don't.  (Not that I expect many people use those but still.)